pkg/agent/route/route_linux.go

```diff
@@ -670,14 +801,121 @@ func (c *Client) syncIPTables() error {
 	return nil
 }
 
+func (c *Client) generateProxyAllRules(nodePortIPSet string,
+	externalIPIPSet string,
+	serviceCIDR string,
+	nodePortDNATVirtualIP string,
+	serviceVirtualIP string,
+	isIPv6 bool) map[string]map[string][]string {
+	rules := map[string]map[string][]string{
+		iptables.NATTable: make(map[string][]string),
+		iptables.RawTable: make(map[string][]string),
+	}
```

Comment:
This nested map is a bit hard to understand and doesn't seem worth. Its consumer also makes assumption on its keys heavily so it's not really generic. I think it's simpler and more readable to just return a natRules and rawRules.

Reply:
Done as you said.